apps/api/routes/extract.py

```
@@ -0,0 +1,261 @@
+
+        # Stub DocumentStore for now
+        # TODO: Replace with actual PostgreSQL adapter
+        from uuid import UUID
+
+        from packages.core.use_cases.extract_pending import DocumentStore
+        from packages.schemas.models import Document
+
+        class StubDocumentStore(DocumentStore):
+            """Stub DocumentStore for MVP."""
+
+            def query_pending(self, limit: int | None = None) -> list[Document]:
```

Ruff (0.14.1)

Unused method argument: `limit`

(ARG002)